usuarios/delete-usuarios.spec.js

```diff
@@ -0,0 +1,31 @@
+const user = require('../utils/userData')
+const testServer = require('../utils/testServer')
+const rota = require('../utils/rotas')
+
+var usuario
+var user_id
```

Comment:
```suggestion
let usuario
let user_id
```

faz mais sentido usar o let
ref https://imasters.com.br/front-end/let-const-e-var-nao-e-tudo-mesma-coisa